pysmiles/smiles_helper.py

```diff
@@ -365,34 +367,39 @@ def valence(atom):
     Returns
     -------
     list[int]
-        The valences for the given atom.
+        The valences for the given atom. Returns an empty list if we don't know.
     """
     element = atom.get("element", '*')
     if element == '*':
-        electrons = 0
-    else:
-        electrons = PTE[element.capitalize()]['AtomicNumber']
-    electrons -= atom.get('charge', 0)
+        return []
 
-    if electrons < 0:
-        raise ValueError(f"Atom {atom} has a negative number of electrons: {electrons}")
```

Comment:
isn't this stull a possibility after line 377?

Reply:
No, it gets caught by the try/except/raise on 380